sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md

```diff
@@ -20,3 +20,4 @@
 
 ## 1.0.0-preview.2
 
+- Official public preview of Azure.DigitalTwins.Core SDK
```

Comment:
Maybe a link to the readme and sample readme for more information about what is included?

Reply:
Could be under the Added header instead